roles/ara_api/tasks/database_engine/django.db.backends.postgresql.yaml

```diff
@@ -54,5 +57,4 @@
   retries: 3
   delay: 5
   until: _ara_sql_migrations is succeeded
-  register: _ara_sql_migrations
```

Comment:
Had to do a double take on this one, it's because we're registering it twice: OK.

Reply:
Yeah, I missed that too first time around :-)